dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+const $tasklist = document.querySelector('.task-list');
+
+$tasklist.addEventListener('click', completeTask);
+
+function completeTask(event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+
+  if (event.target && event.target.tagName === 'BUTTON') {
+    const closest = event.target.closest('.task-list-item');
```

Comment:
Don't forget the `$` in front of the variable name! You're storing a DOM element, so we should include the dollar sign